charts/stable/homepage/values.yaml

```
@@ -1,6 +1,6 @@
 image:
-  repository: tccr.io/truecharts/homepage
-  tag: v0.7.1@sha256:f51e1a40b0c683615bcdcf12b2632517a32800b0875f2d64c3a3cb5c4493fb5e
+  repository: ghcr.io/gethomepage/homepage
```

```suggestion
  repository: tccr.io/truecharts/homepage
```
https://github.com/truecharts/containers/blob/master/mirror/homepage/Dockerfile

Don't change the container repo, we're already using the proper repo upstream